### PR TITLE
Advanced send options

### LINKED ISSUE #3045: Advanced Send Options
Set these options for current transaction:

* Fee policy
* Spend unconfirmed transactions

This action doesn't store the selected option in **preferences**.

### LINKED ISSUE #3075: ChromeApp can not recreate any wallet
Error with: `window.alert()` that is not allowed in ChromeApp.

```
Client ERROR: Copayer not found
extensions::platformApp:17 window.alert() is not available in packaged apps.
```

### PR DESCRIPTION
**Pull request by cmgustavo**
fixes #3045
fixes #3075

### CODE REVIEW

**Comment by matiu** [comment]
code LGTM, will try it later today.

File: public/views/walletHome.html [hunk 1]
```diff
@@ -412,6 +412,33 @@ <h4 class="title m0">
...
+                <h4 class="title m0" translate>Fee policy for this transaction</h4>
+                <ul class="no-bullet m0 size-14">
+                  <li ng-repeat="fee in (index.network == 'livenet' ? index.feeLevels.livenet : index.feeLevels.testnet)"
+                    ng-click="home.currentSendFeeLevel = fee.level" class="line-b p20">
+                    {{index.feeOpts[fee.level]|translate}}
+                    <i class="fi-check size-16 right" 
+                      ng-show="(home.currentSendFeeLevel || index.currentFeeLevel) == fee.level"></i>
+                  </li>
+                </ul>
+                <h4 class="title m0" translate>Unconfirmed transactions for this transaction</h4>
```
Comment (dabura667) [inline public/views/walletHome.html]: This is confusing, maybe "Unconfirmed outputs for this transaction"

**Comment by dabura667** [comment]
tbh "Spending a transaction" will probably confuse users... (not that "spending an output" is any less confusing...

However, this is "Advanced" so I think we should be specific and say "Ouputs" instead of "Transaction."

you spend outputs, not transactions.

File: public/views/walletHome.html (hunk 1 above)
Comment (isocolsky) [inline public/views/walletHome.html]: I think we should assess how intuitive the concept of "spending unconfirmed transactions" is to the end user. Maybe we should change it to "spend unconfirmed funds"?

Comment (dabura667) [inline public/views/walletHome.html]: I agree. "Unconfirmed funds" sounds reasonable.

**Comment by cmgustavo** [comment]
Updated. Thanks for your feedback!

File: public/views/walletHome.html [hunk 2]
```diff
@@ -412,6 +412,33 @@ <h4 class="title m0">
...
+                  <li ng-repeat="fee in (index.network == 'livenet' ? index.feeLevels.livenet : index.feeLevels.testnet)"
+                    ng-click="home.currentSendFeeLevel = fee.level" class="line-b p20">
+                    {{index.feeOpts[fee.level]|translate}}
+                    <i class="fi-check size-16 right" 
+                      ng-show="(home.currentSendFeeLevel || index.currentFeeLevel) == fee.level"></i>
+                  </li>
+                </ul>
+                <h4 class="title m0" translate>Use unconfirmed funds for this transaction</h4>
+                <div class="p20 line-b">
+                  <span class="size-14" translate>Use Unconfirmed Funds</span>
```
Comment (isocolsky) [inline public/views/walletHome.html]: One last comment, could you please not capitalize every word. I think "Use unconfirmed funds" is easier to read.

File: public/views/walletHome.html [hunk 3]
```diff
@@ -412,6 +412,33 @@ <h4 class="title m0">
               </div>
             </div>
 
+            <div class="m20b" ng-init="home.hideAdvSend=true">
+              <a class="button outline light-gray expand tiny m0" ng-click="home.hideAdvSend=!home.hideAdvSend">
+                <i class="fi-widget m3r"></i>
+                <span translate ng-hide="!home.hideAdvSend">Show Advanced options</span>
+                <span translate ng-hide="home.hideAdvSend">Hide Advanced options</span>
```
Comment (isocolsky) [inline public/views/walletHome.html]: Same here with "advanced" instead of "Advanced"

**Comment by cmgustavo** [comment]
Updated. Thanks @matiu

**Pull request merged**